CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Unreleased
 ------
 
+Version 1.8.0
+-------------
+
+* [1119](https://github.com/Shopify/shopify-app-cli/pull/1119): Update EP version and enable guest serialization for scripts
```

Comment:
I'm not sure I fully understand what this changelog entry means, could it be simplified to `enable guest serialization for scripts`?

Reply:
Also, consider if we need to provide further context here. Think of it from the perspective of a third-party developer upgrading from an older shopify_app, who is reading the changelog to understand if they need to change anything in their app.

Reply:
@andyw8 Sorry, just noticed your comment from my email notification. So far, script is still mostly used internally. We simply want a version bump which can be pointed to for the script users. But yeah, your comments make sense. I'll keep that in mind for the future release.